core/controllers/story_editor.py

```diff
@@ -200,6 +244,26 @@ class ValidateExplorationsHandler(base.BaseHandler):
     """A data handler for validating the explorations in a story."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'story_id': {
+            'schema': {
+                'type': 'basestring'
+            },
+            'validators': [{
+                'id': 'has_length',
+                'value': constants.STORY_ID_LENGTH
+            }]
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'comma_separated_exp_ids': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
+        }
+    }
```

Comment:
Can you use `JsonEncodedInString` here?

Reply:
Is this referring to the schema type? I'm unable to find a matching type for `JsonEncodedInString` in `schema_utils.py`.

Reply:
It is in `extensions/objects/models/objects.py` you can look at how it is used by searching for `JsonEncodedInString` in the codebase.

Reply:
I tested with `JsonEncodedInString` but I'm not sure if the type applies here. I looked at other instances that used the schema definition and they were all lists. `comma_separated_exp_ids` has an additional parsing step for the delimiter:
`        topic_ids = comma_separated_topic_ids.split(',')
`

Reply:
Yep, you will need to change the tests and the frontend to actually use JSON instead of comma-delimited list.

Reply:
I'm more than happy to try my hand at that, but it seems a bit out of scope for this pull request and linked issue fix. Could this be made into a new issue and assigned separately?

Reply:
I think it is in the scope, the main purpose why we introduce the schema validation is to unify stuff like this.

Reply:
I don't think you need to change any other backend files, for reference you can look at this PR #14563